### ISSUE TITLE
[form][field]  `.formattedValue` should be set to undefined when not being able to format

### ISSUE THREAD

**Issue by tlouisse** [ing-bank/lion#129]
Currently, this happens:

```js
        if (this.modelValue instanceof Unparseable) {
          return this.modelValue.viewValue;
        }
        if (this.errorState) {
          return this.inputElement ? this.value : undefined;
        }
        return this.formatter(this.modelValue, this.formatOptions);
      }
```
https://github.com/ing-bank/lion/blob/master/packages/field/src/FormatMixin.js#L186

It should not return the current value of the inputElement and sync it back. Whenever the formattedValue returns undefined, it should be decided in the sync down step to leave inputElement.value as is

**Comment by daKmoR** [comment]
I added a failing test... (pushed a commit)
```js
  it('leaves partial (and therefore unparseable) inputs untouched', async () => {
    const el = await fixture(html`
      <lion-input-date .value=${'2017/06'}></lion-input-date>
    `);
    expect(el.value).to.equal('2017/06');
  });
```

**Comment by jorenbroekema** [comment]
Could you confirm whether this issue is still open @tlouisse  ?

**Comment by tlouisse** [comment]
Yep, it should just return `undefined` when invalid(errorState) , and leave `_inputNode.value` untouched

**Comment by daKmoR** [comment]
Task to do:

```js
<lion-date-input>
el.modelValue = 'foo';
```

current:
```js
el.formattedValue // ''
el.value // ''
```

wanted:
```js
el.formattedValue // undefined
el.value // ''
```

**Comment by tlouisse** [comment]
The idea was to make `formattedValue` "conceptually consistent".
When formatting didn't take place, this would be resembled by `undefined` (this means current modelValue can't be formatted). 
As a consequence, the formattedValue would not be synced to `_inputNode.value` (== viewValue). (a redundant step)
 
I changed my mind on this topic for a few reasons:
- If we have an Unparseable modelValue, we also set formattedValue to modelValue.viewValue (this is needed, because _inputNode.value might be empty or different). 
- formattedValue can be read via `myForm.formattedValue` to get all (scheduled) viewValues <= important one as we noticed recently 

If we think of `.formattedValue` as the 'scheduled viewValue'(after blur by default), it actually makes sense to store the current viewValue in the formattedValue.

So my notion of "conceptually consistent" changed and if it were up to me, we can close this issue and keep the current behavior :)
